core/services/functions/connector_handler_test.go

```diff
@@ -80,6 +80,7 @@ func TestFunctionsConnectorHandler(t *testing.T) {
 	subscriptions.On("Start", mock.Anything).Return(nil)
 	subscriptions.On("Close", mock.Anything).Return(nil)
 	config := &config.PluginConfig{
+		// undefined: gwconnector.ConnectorConfigcompilerUndeclaredImportedName
```

Comment:
It's looking for what you moved as core/config ConnectorGatewayConfig, but as a struct